modules/ossm-tutorial-bookinfo-verify-install.adoc

```diff
@@ -13,12 +13,11 @@ To confirm that the sample Bookinfo application was successfully deployed, perfo
 .Prerequisites
 
 * {SMProductName} {SMProductVersion} installed.
-* Access to the OpenShift CLI (`oc`).
 * Complete the steps for installing the Bookinfo sample app.
 
-.Procedure
+.Procedure from CLI
 
-. Log in to the {product-title} CLI.
+. Log in to the CLI.
```

Comment:
This is a bit of a nitpick, but one of the reasons why I always write "{product-title} CLI" is to reinforce that Service Mesh does not support the Istio CLI (`istioctrl`).

Reply:
Done